src/components/EngineManageDialog.vue

```diff
@@ -421,12 +421,18 @@ watch(
       const version = await store
         .dispatch("INSTANTIATE_ENGINE_CONNECTOR", { engineId: id })
         .then((instance) => instance.invoke("versionVersionGet")({}))
+        .then((version) => {
+          // OpenAPIのバグで"latest"のようにダブルクォーテーションで囲まれていることがあるので外す
+          if (version.startsWith('"') && version.endsWith('"')) {
+            return version.slice(1, -1);
```

Comment:
```suggestion
            const parsedVersion = JSON.parse(version);
            if (typeof parsedVersion !== "string")
            	throw new Error("typeof JSON.parse(version) !== string")
            return parsedVersion
```
バックスラッシュとかも処理してくれるJSON.parseの方がいい気がしました。
```
> JSON.parse('"Backslash \\""')
'Backslash "'
```

Reply:
@sevenc-nanashi お、もしかしてバックスラッシュも入ってた感じでしょうか･･･？

Reply:
いや、入ってはいませんでした。
が、`""`で囲まれてる=JSONの文字列がそのまま返ってきている（=中の`"`がエスケープされてる）という解釈なのでJSON.parseしたい、という感じです。

Reply:
なるほどです。やるならtry catchでくるんでJSON.parseし、stringになったかの判定とかですかね。
現状だとそのケースが確認されていないならいったんそのままでも大丈夫ですかね！